pom.xml

```diff
@@ -116,75 +138,51 @@
-                <artifactId>maven-shade-plugin</artifactId>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <version>0.13</version>
                 <executions>
                     <execution>
-                        <id>assemble-org.killbill.billing.plugin.catalog.test</id>
                         <goals>
-                            <goal>shade</goal>
+                            <goal>check</goal>
                         </goals>
-                        <phase>package</phase>
+                        <phase>verify</phase>
                         <configuration>
-                            <createSourcesJar>true</createSourcesJar>
-                            <shadedArtifactAttached>false</shadedArtifactAttached>
-                            <shadedClassifierName>jar-with-dependencies</shadedClassifierName>
-                            <filters>
-                                <filter>
-                                    <artifact>${project.groupId}:${project.artifactId}</artifact>
-                                </filter>
-                            </filters>
+                            <skip>${check.skip-rat}</skip>
+                            <ignoreErrors>${check.ignore-rat}</ignoreErrors>
+                            <licenseFamilies>
+                                <licenseFamily implementation="org.apache.rat.license.Apache20LicenseFamily"/>
+                                <licenseFamily implementation="org.apache.rat.license.MITLicenseFamily"/>
+                            </licenseFamilies>
+                            <useDefaultExcludes>true</useDefaultExcludes>
+                            <parseSCMIgnoresAsExcludes>true</parseSCMIgnoresAsExcludes>
+                            <licenses>
+                                <license implementation="org.apache.rat.analysis.license.SimplePatternBasedLicense">
+                                    <licenseFamilyCategory>MIT</licenseFamilyCategory>
+                                    <licenseFamilyName>The MIT License</licenseFamilyName>
+                                    <patterns>
+                                        <pattern>@license MIT</pattern>
+                                        <pattern>Dual licensed under the MIT and GPL licenses.</pattern>
+                                        <pattern>License: MIT</pattern>
+                                        <pattern>MIT License</pattern>
+                                        <pattern>MIT/GPL2 Licensed</pattern>
+                                        <pattern>jquery.org/license</pattern>
+                                        <pattern>This file is generated by jOOQ.</pattern>
+                                    </patterns>
+                                </license>
+                            </licenses>
+                            <excludes combine.children="append">
```

Comment:
You don't need to copy the whole thing, that's why we have `combine.children="append"` 😄 

Check https://github.com/killbill/killbill-avatax-plugin/blob/8e71564fd5979829725012971b6f6fe5c2ad4864/pom.xml#L211

Reply:
I find it really weird that this `append` property which is set for the [excludes list](https://github.com/killbill/killbill-oss-parent/blob/master/pom.xml#L2261), would apply for other things outside of such exclude list. I did not repeat all the excludes as per property, but then how much more can I remove - everything the `configuration` above?

Reply:
Yup, just copy the Avatax example above.

https://blog.sonatype.com/2011/01/maven-how-to-merging-plugin-configuration-in-complex-projects/